ZupIT/ritchie-cli
# Adding --force-build flag

Signed-off-by: Gabriel Pinheiro <gabrielctpinheiro@gmail.com>



### Description
You can now run formulas with a `--force-build` (or `-b`) flag that will force it to be built (ok haha) regardless of changes without prompting the user with an extra question

### How to verify it
Run any formula that exists locally in a repository with --force-build or -b and check the build spinner before the execution

### Changelog
Added `--force-build` (or `-b`) flag

![force-build](https://user-images.githubusercontent.com/56726395/99318908-5d6ac700-2847-11eb-8aca-946d641d6afa.gif)

Closes #668
